MemoryManagement/src/memorymanagementAlgorithm/Process.java

```diff
@@ -1,140 +1,192 @@
 package memorymanagementAlgorithm;
 
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+/**
+ *
+ * @author ahmed
+ */
+
 import java.util.*;
 import javafx.scene.paint.Color;
+//Vector<Double> list = new Vector<Double>();
 
 public class Process {
 
+    
+
+    private Color color;
     private static int Process_ID = 0;
     private int ID;
-    private Color color;
     private boolean inserted = false;
+    private int number_of_segments;
     private Vector<Segment> Segment_vector;
-
     //====================constructor============================//
-    public Process() {
+
+    public Process()// just the id

```

Comment:
never put a comment like this between function name and its start bracket. So the auto-formatting can committed.

Reply:
in c++ this show the comment on when you use the function but it dosen't work here gonna fix it ./
not fixed .